plugin/internal/remotecall/types.go

```diff
@@ -0,0 +1,14 @@
+package remotecall
```

Comment:
should we still add licenses?

Reply:
dont know which one to add - they also have many files without licenses